.github/workflows/test-package.yml

```diff
@@ -21,6 +21,9 @@ jobs:
   # against Dart stable.
   analyze:
```

Comment:
We will have two sub-packages jni_gen and jni.
We should analyze and test both.

So we should either (a) call this analyze_jni_gen or (b) move the `working-directory` to each run step, so we can have a different working directory for analyzing jni.

Though we can also address this with the PR adding `jni`.

Reply:
working-directory is per-job it seems. I think I will add an analyze_jni in the commit adding jni. jni_gen is the main part anyway.

Reply:
Nvm. Renamed it for uniformity.